client/app/src/accounts/account.controller.js

```diff
@@ -120,7 +120,9 @@
     ]
 
     // 1 ARK has 100000000 "arkthosi"
-    const UNIT = Math.pow(10, 8)
+    const ARKTOSHI_VAL = Math.pow(10, 8)
+
+    self.ARKTOSHI = ARKTOSHI_VAL
```

Comment:
I'm thinking this should belong in a different file for constants

Reply:
Wasn't entirely sure where to put the constant. Where do you think it should be?

I put it in the AccountController since all the html files w/ the value used that controller. So it minimized the scope.

Reply:
Maybe create `client/app/config/constants.js` or something to that affect? I did notice UNIT had been declared in a few controllers, so it can replace those too.